src/graphql/schema/buildVersionType.ts

```diff
@@ -0,0 +1,19 @@
+import { GraphQLBoolean, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { DateTimeResolver } from 'graphql-scalars';
+import formatName from '../utilities/formatName';
+
+const buildVersionType = (type: GraphQLObjectType): GraphQLObjectType => {
+  return new GraphQLObjectType({
+    name: formatName(`${type.name}Version`),
+    fields: {
+      id: { type: GraphQLString },
+      parent: { type: GraphQLString },
+      autosave: { type: GraphQLBoolean },
```

Comment:
only add `autosave` if the option is enabled on the collection. it should not be added 100% of the time